package.json

```diff
@@ -18,6 +18,7 @@
     "babel-core": "^6.18.2",
     "babel-loader": "^6.2.7",
     "babel-plugin-module-resolver": "^2.3.0",
+    "babel-preset-env": "^1.6.1",
```

Comment:
Yeah! That would be great. If you have the time to do that, I'd appreciate that. You can push that onto this branch or master and I can get those changes from there. Thanks!

Reply:
@tildeee Ok once things are ready, just ping me and I'll migrate things to WP3.

Reply:
Not sure how much more of a pain it would be, but I think that it might be easier to ensure a cohesive result if we rebuilt this project on top of the new baseline rather than tried to integrate that stuff onto what we already have here.